### PR TITLE
update CHANGELOG.md

### PR DESCRIPTION
**Pull request by rfomin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,36 @@
...
-* Rename `-nodehlump` to `-nodeh` for consistency with other ports.
-* Use sector lightlevel for sprites in Boom and Vanilla complevels.
+* 3D audio support (@ceski-1)
+  - HRTF mode ("Headphones mode" in the General menu).
+  - Air absorption and Doppler effects settings in the config.
+* PC Speaker emulation sound module (taken from Chocolate Doom).
+* Various HUD additions and fixes:
+  - Optional widescreen widget arrangement.
+  - Bring back three-lined coords/stats widgets.
+  - Optionally draw bar graphs in Boom HUD widgets.
```
Comment (rfomin) [inline CHANGELOG.md:9]: @fabiangreffrath I don't really understand what exactly is fixed/added to the HUD, there seems to be a lot of changes! Please give your input on this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,36 @@
 **New Features and Improvements**
-* Add direct aiming from Crispy Doom. More accurate vertical aiming (thanks to @ceski-1).
-* Optimizations for plane rendering, taken from Eternity Engine (thanks to @JNechaevsky).
-* Add `-dedicated`, `-uncapped` and `-nouncapped` command line parmeters (thanks to @loopfz).
-* Rename `-nodehlump` to `-nodeh` for consistency with other ports.
-* Use sector lightlevel for sprites in Boom and Vanilla complevels.
+* 3D audio support (@ceski-1)
+  - HRTF mode ("Headphones mode" in the General menu).
```
Comment (ceski-1) [inline CHANGELOG.md:3]: Small edit:
```
- Stereo and up to 7.1 surround sound.
- HRTF support ("Headphones mode" in the General menu).
```

Comment (rfomin) [reply CHANGELOG.md:4]: Done in https://github.com/fabiangreffrath/woof/pull/1208/commits/0a56da97df9c2860260b99c273c5b1bf6e2c47be

File: CHANGELOG.md (hunk 1 above)
Comment (fabiangreffrath) [reply CHANGELOG.md:10]: Ability to position "message" and "secret" text widgets by WOOFHUD lump.

Comment (rfomin) [reply CHANGELOG.md:10]: Done in https://github.com/fabiangreffrath/woof/pull/1208/commits/465b0e9c0aa1bad60b27c9a7e590a9538aa78e2b

**Review by fabiangreffrath — APPROVED** [review]

**Pull request merged**